contribs/freight/src/main/java/org/matsim/freight/carriers/analysis/CarrierPlanAnalysis.java

```diff
@@ -54,103 +55,149 @@ public class CarrierPlanAnalysis {
 	private static final Logger log = LogManager.getLogger(CarrierPlanAnalysis.class);
 	public final String delimiter;
 
+	public enum CarrierAnalysisType {onlyDemand, completeAnalysis}
```

Comment:
nice to have this enum!
Should it be better: onlyCarrier, analysisIncludingEvents or similiar... (I have not a perfect idea now)
onlyDemand (jobs?) and completeAnalysis does not sound perfectly clear for me, what kind of input is needed.

Reply:
renamed and move enum